packages/nextjs/components/impact-vector/ImpactVectorCard.tsx

```diff
@@ -4,20 +4,23 @@ import React from "react";
 import Image from "next/image";
 import { useRouter } from "next/navigation";
 
-const ImpactVectorCard = () => {
+interface ImpactVectorCardProps {
+  name: string;
+  description: string;
+  username: string;
+}
+const ImpactVectorCard = ({ name, description, username }: ImpactVectorCardProps) => {
   //the route is hard coded for now
   const router = useRouter();
   return (
     <div
       onClick={() => router.push("/impact/1")}
       className="cursor-pointer rounded-xl text-sm border-[0.2px] border-secondary-text/50 p-4 bg-base-300 flex flex-col justify-between gap-4"
     >
-      <h2 className="pl-2  m-0 font-bold">Ecosystem users</h2>
+      <h2 className="pl-2  m-0 font-bold">{name}</h2>
```

Comment:
Let's trim the "OSO: " from the beginning of each name here where it is displayed.